.github/workflows/update-intellij-version.yml

```
@@ -0,0 +1,46 @@
+name: Update IntelliJ Version
+
+on:
+  pull_request:
+
+jobs:
+  bump-intellij-version:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.3.4
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Update default IntelliJ version
+        run: |
```

```suggestion
        # language=sh
        run: |
```
This should highlight the script as a shell script injection in the IntelliJ (assuming IntelliLang plugin is enabled)

---

(thus also highlighting certain problems, like quoting issues)